bin/lambda-local

```diff
@@ -52,6 +52,8 @@
 		awsRequestId: _context.createInvokeId,
 		timeoutMs: timeoutMs
 	});
+    // export the LAMBDA_TASK_ROOT enviroment variable
+    process.env['LAMBDA_TASK_ROOT'] = process.cwd();
```

Comment:
`process.cwd()` gives you the current directory, so it will change depending on where you are when executing the script. I cannot find official documentation for `LAMBDA_TASK_ROOT`, but I am assuming that it is supposed to give you the directory where the lambda script is located. 

I believe 
```
process.env['LAMBDA_TASK_ROOT'] = __dirname;
```

Is what you are looking for.